### ISSUE TITLE
Train takes too many epochs long for en_PP-OCRv3_rec  config

### ISSUE THREAD

**Issue by Arminkhayati** [PaddlePaddle/PaddleOCR#8992]
Hi 
I am using the config specified [here ](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/rec/PP-OCRv3/en_PP-OCRv3_rec.yml) (PP-OCRv3_rec) for id card images OCR.
Model architecture related configs are the same. I only introduced my dataset which is a LMDB dataset in the config file and changed batch size to 32. That's the only edit I made in the config.

The problem is it takes too long to learn. I did the sanity check with only 1 image (Trained model with only one image), it learns the image in 250 to 300 epochs while [this config](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/configs/rec/rec_svtrnet.yml) learns it in 25 epochs.
I insist on using the first config, but I don't know if it is normal to take this long to train. 
Also I tried different learning rate values, hidden dimension sizes but it doesn't make any difference in a good way.

- System Environment：Ubuntu 20.04 LTS
- Version：
paddle-bfloat:  0.1.7
paddlepaddle-gpu:2.4.1 

I

**Comment by andyjpaddle** [comment]
the training time depends on the amount of your dataset and the epochs, you can finetune your model on your dataset by using pretrained model, this will speed up training. you can refer to this doc https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/doc/doc_en/finetune_en.md#3-text-recognition-model-fine-tuning

**Comment by Arminkhayati** [comment]
> the training time depends on the amount of your dataset and the epochs, you can finetune your model on your dataset by using pretrained model, this will speed up training. you can refer to this doc https://github.com/PaddlePaddle/PaddleOCR/blob/dygraph/doc/doc_en/finetune_en.md#3-text-recognition-model-fine-tuning

Thank you.
As I mentioned above, I did sanity check with only one image (one data sample) and it takes 300 epochs to learn only one sample, while the other config I linked above, takes only 20-25 epochs too learn it.
For 10 images (10 data sample) it takes 3000 epochs.
My dataset consist of 59000 images. Then imagine how long it takes to learn it.

**Comment by andyjpaddle** [comment]
you can set epochs by your own, usually , the more dataset, the small epoch
